overcooked_ai_py/mdp/overcooked_mdp.py

```diff
@@ -1551,13 +1551,15 @@ def state_string(self, state):
     def lossless_state_encoding_shape(self):
         return np.array(list(self.shape) + [20])
 
-    def lossless_state_encoding(self, overcooked_state, debug=False):
+    def lossless_state_encoding(self, overcooked_state, horizon, debug=False):
```

Comment:
I think we might want to make `horizon` a kwarg, i.e. `horizon=None` or `horizon=400` so this is backwards compatible

Reply:
OK, changed